obspy/taup/tau.py

```diff
@@ -379,6 +391,170 @@ def get_ray_paths(self, source_depth_in_km, distance_in_degree=None,
         return Arrivals(sorted(rp.arrivals, key=lambda x: x.time),
                         model=self.model)
 
+    def get_travel_times_geo(self, source_depth_in_km, source_latitude_in_deg,
+                             source_longitude_in_deg, receiver_latitude_in_deg,
+                             receiver_longitude_in_deg, phase_list=("ttall",)):
+        """
+        Return travel times of every given phase given geographical data.
+
+        .. note::
+
+        Note that the conversion from source and receiver latitudes and
```

Comment:
This needs to be indented to be in the note.

PS, there are no notifications when you push updates, so you need to ping the ticket if you want it re-reviewed.